src/components/vanilla/charts/TableChart/TableChart.emb.ts

```diff
@@ -74,6 +75,20 @@ export const meta = {
         label: 'Show download as CSV',
         category: 'Export options',
         defaultValue: true
+    },
+    // Table styling
+    {
+      name: 'minColumnWidth',
+      type: 'number',
+      label: 'Minimum metric column width in pixels',
```

Comment:
@kuka-radovan please could you remove the word 'metric' here, since it refers to all columns?

Reply:
Done